### PR TITLE
[DC-1851] Modify EHR union such that the excluded sites’ tables are not part of the UNION queries

### PR DESCRIPTION
**Pull request by ratuagga**
Updates added for excluding Hpo_id list instead of including hpo_ids for processing.

### CODE REVIEW

File: tests/integration_tests/data_steward/validation/ehr_union_test.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 PITT_HPO_ID = 'pitt'
 NYC_HPO_ID = 'nyc'
+THIRD_HPO_ID = 'fake'
```
Comment (nishanthpp93) [inline tests/integration_tests/data_steward/validation/ehr_union_test.py:23]: Can you name this `EXCLUDED_HPO_ID`?
```suggestion
EXCLUDED_HPO_ID = 'excluded'
```

File: tests/integration_tests/data_steward/validation/ehr_union_test.py [hunk 2]
```diff
@@ -78,7 +79,7 @@ def _create_hpo_table(self, hpo_id, table, dataset_id):
 
     def _load_datasets(self):
         """
-        Load five persons data for each test hpo
+        Load five persons data for nyc and pitt test hpo and rdr data for the Fake_hpo
```
Comment (nishanthpp93) [inline tests/integration_tests/data_steward/validation/ehr_union_test.py:82]:
```suggestion
        Load five persons data for nyc and pitt test hpo and rdr data for the excluded hpo
```

**Review by nishanthpp93 — CHANGES_REQUESTED** [review]
I think the changes look good, added a naming suggestion to make it more evident.

File: tests/integration_tests/data_steward/validation/ehr_union_test.py [hunk 3]
```diff
@@ -141,7 +149,14 @@ def _dataset_tables(self, dataset_id):
         tables = bq_utils.list_tables(dataset_id)
         return [table['tableReference']['tableId'] for table in tables]
 
-    def test_union_ehr(self):
+    @mock.patch('bq_utils.get_hpo_info')
+    def test_ehr_main_exclude_hpo_id(self, mock_hpo_info):
+        mock_hpo_info.return_value = [{
+            'hpo_id': hpo_id
+        } for hpo_id in self.hpo_ids]
```
Comment (nishanthpp93) [inline tests/integration_tests/data_steward/validation/ehr_union_test.py:156]: This looks like it is not testing anything. You can remove it.

**Review by nishanthpp93 — CHANGES_REQUESTED** [review]
Thanks for the update. Looks good except for the one unused test.

File: tests/integration_tests/data_steward/validation/ehr_union_test.py (hunk 3 above)
Comment (ratuagga) [reply tests/integration_tests/data_steward/validation/ehr_union_test.py]: done

**Review by nishanthpp93 — APPROVED** [review]

**Pull request merged**